salt/_roster/kubernetes_nodes.py

```diff
@@ -57,5 +57,6 @@ def targets(tgt, tgt_type='glob', **kwargs):
                 'user': annotations.get(prefix + 'user', 'root'),
                 'priv': annotations.get(prefix + 'key-path', 'salt-ssh.rsa'),
                 'sudo': bool(annotations.get(prefix + 'sudo', False)),
+                'minion_opts': {'use_superseded': ['module.run']},
```

Comment:
Do we need it in here if it's already a default value, see below?

Reply:
Here, no we don't need it,
But if one day we would like to customize some others minion_opts here, that's overwrite all the value "minion_opts" so I prefer have it in both so if someone want to use another roster generated manually or if one day we add some minion_opts in the roster we have it